inference-engine/src/cldnn_engine/cldnn_engine.cpp

```diff
@@ -308,18 +308,6 @@ InferenceEngine::CNNNetwork clDNNEngine::CloneAndTransformNetwork(const Inferenc
                     return true;
                 });
 
-            pass_config->set_callback<ngraph::pass::ConvertNMS1ToNMS5,
```

Comment:
because we added callback to `ConvertNMSToNMSIEInternal` for CPU to avoid conversion `NMS-5` to `NMSIEInternal` if `NMS-5` last node in plug-in. after which GPU tests for NMS-5  broke, because before it `ConvertNMSToNMSIEInternal`,  didn't contain callback call inside transformation.
`ConvertNMS1ToNMS5, ConvertNMS3ToNMS5, ConvertNMS4ToNMS5` still don't contain callback call inside transformation.
so looks like callback registration was useless.

@vladimir-paramuzov could you take a look at this please?

Reply:
I'm OK with this change.